src/renderer/components/AccountView.tsx

```diff
@@ -52,6 +55,66 @@ function AccountView(props: { pubKey: string | undefined }) {
     undefined
   );
 
+  // create dummy keypair wallet if none is selected by user
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I don't like this, but meh.

I _think_ I'd rather we find a way to have the electron wallet auto select itself without user interaction 

(I hope you're using it....)

Reply:
When you say don't like this, are you talking about the dummy keypair wallet, or the exhaustive deps exemption?

The problem I ran into with exhaustive deps in this case was that the Anchor provider wants to take in a wallet -- which, generally would be the one selected by Workbench. Accessing this wallet is easy using `useAnchorWallet`, but because that is a hook, it can't be nested inside or used inside other hooks like the `useEffect` in this case. I guess we could store it in state? But disabling the lint issue works for now, and should still re-run the hook if a new wallet is selected (I'll have to confirm w/ testing).